notifications/idlharness.any.js

```diff
@@ -0,0 +1,25 @@
+// META: global=window,worker
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+// https://notifications.spec.whatwg.org/
+
+idl_test(
+  ['notifications'],
+  ['service-workers', 'html', 'dom'],
+  idl_array => {
+    idl_array.add_objects({
+      Notification: ['notification'],
+      NotificationEvent: ['notificationEvent'],
+    });
+    if (self.isWorker) {
```

Comment:
This is probably a typo of `self.GLOBAL.isWorker()`, but in any case https://w3c.github.io/ServiceWorker/#serviceworkerglobalscope-interface is `Exposed=ServiceWorker`, so this should be testing for service workers only. I'll tweak it.